apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -725,8 +793,23 @@ properties:
     constants: Titanium.Media.VIDEO_SCALING_*
     default: <Titanium.Media.VIDEO_SCALING_ASPECT_FIT>
 
+  - name: showsControls
+    summary: Whether or not the receiver shows playback controls. Default is YES.
+    description: |
+        Clients can set this property to NO when they don't want to have any playback controls on top of 
+        the visual content (e.g. for a game splash screen). This property should not be used to temporarily
+         change the visibility of the playback controls since it will create or destroy UI elements.    
+    platforms: [iphone, ipad]
+    type: Boolean
+    default: true
+    since: "7.0.0"
+
   - name: sourceType
-    summary: The playback type of the movie.
+    deprecated:
+        since: "7.0.0"
+        notes: This method has been removed in Titanium SDK 7.0.0 as of the official deprecation by Apple.
```

Comment:
Add: `The source type is detected automatically in the newer Apple API's used.` Full:
```yml
notes: |
    This method has been removed in Titanium SDK 7.0.0 as of the official deprecation by Apple.
    The source type is detected automatically in the newer Apple API's used.
```